app/src/main/java/org/oppia/app/topic/TopicTab.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.topic
+
+/** Enum to store the tabs of topic and get tab by position. */
```

Comment:
Update this comment to 
`/** Enum to store the tabs of [TopicFragment] and get tab by position. */`

Topic is a very generally thing which might reference to the topic coming from oppia-backend, domain module, etc. but these tabs are actually specifically related to `TopicFragment file`.

Reply:
Used the suggested comment